### PR TITLE
Add timeline component in contact details page

### LINKED ISSUE #1047: Migrate timeline component in contact details page from contacts app

### PR DESCRIPTION
**Pull request by edifirut**

### CODE REVIEW

File: packages/apps/spaces/components/ui-kit/atoms/timeline-item/TimelineItem.tsx [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-          {/*  date={createdAt}*/}
-          {/*  format={'D-M-YYYY h:mm A'}*/}
-          {/*></Moment>*/}
+          <ReactTimeAgo
+            className='text-sm text-gray-500 mb-1'
+            date={new Date(createdAt)}
+            locale='en-US'
+          />
+          <Moment
+            className='text-sm text-gray-500'
```
Comment (Kasia9090) [inline packages/apps/spaces/components/ui-kit/atoms/timeline-item/TimelineItem.tsx]: Use date-fns instead

File: packages/apps/spaces/package.json [hunk 2]
```diff
@@ -45,6 +45,10 @@
     "react-toastify": "^9.1.1",
     "react-use-websocket": "^4.2.0",
     "react-virtual": "^2.4.10",
+    "react-time-ago": "^7.2.1",
+    "react-moment": "^1.1.3",
```
Comment (Kasia9090) [inline packages/apps/spaces/package.json]: moment is slow and heavy, let's use date-fns instead, it's already in the dependencies

File: packages/apps/spaces/pages/_app.tsx [hunk 3]
```diff
@@ -23,10 +23,14 @@ import { RecoilRoot } from 'recoil';
 //   })
 // }
 
+import TimeAgo from 'javascript-time-ago';
+import en from 'javascript-time-ago/locale/en.json';
```
Comment (Kasia9090) [inline packages/apps/spaces/pages/_app.tsx]: Not needed, date-fns can deal with it

File: packages/apps/spaces/components/contact/contact-history/ContactHistory.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    loading: notesLoading,
+    error: notesError,
+  } = useContactNotes({ id });
+  // const {data: actions, loading: actionsLoading, error: actionsError} = useContactActions({id});
+  // TODO add pagination support
+  const {
+    data: conversations,
+    loading: conversationsLoading,
+    error: conversationsError,
+  } = useContactConversations({ id });
```
Comment (Kasia9090) [inline packages/apps/spaces/components/contact/contact-history/ContactHistory.tsx:19]: [neat pick] those calls could be moved to single hook that returns already sorted items
 (you can ignore it)

File: packages/apps/spaces/components/ui-kit/organisms/timeline/Timeline.tsx [hunk 5]
```diff
@@ -93,14 +96,10 @@ export const Timeline = ({
...
-    <div className='mt-5 mb-3'>
-      <div className='text-sm text-gray-500 flex justify-content-center mb-1'>
-        {' '}
-        Now{' '}
-      </div>
+    <div className='mb-3'>
       {loggedActivities.map((e: any, index) => (
-        <React.Fragment key={e.id}>
-          {getTimelineItemByTime(e.type, e, index)}
+        <React.Fragment key={uuidv4()}>
```
Comment (Kasia9090) [inline packages/apps/spaces/components/ui-kit/organisms/timeline/Timeline.tsx:95]: Why this change?

**Review by Kasia9090 — CHANGES_REQUESTED** [review]

File: packages/apps/spaces/components/ui-kit/organisms/timeline/Timeline.tsx (hunk 5 above)
Comment (edifirut) [reply packages/apps/spaces/components/ui-kit/organisms/timeline/Timeline.tsx:95]: conversations duplicated by id. we'll fix this later when we move to a unified timeline endpoint

File: packages/apps/spaces/components/contact/contact-history/ContactHistory.tsx (hunk 4 above)
Comment (edifirut) [reply packages/apps/spaces/components/contact/contact-history/ContactHistory.tsx:19]: i'll wrap this later when i add back actions

**Review by Kasia9090 — APPROVED** [review]
